docs/website/layouts/support/section.html.html

```diff
@@ -17,25 +17,25 @@ <h1 class="banner-content">
       </div>
 
       <div class="grid-container page-content">
-        <div class="card" onclick="window.open('https://www.styra.com/styra-opa-support?utm_medium=partner&utm_source=opa&utm_campaign=dmge&utm_content=opa-support')">
```

Comment:
This will make only part of the card clickable, instead of the complete thing, right?
Should we instead add `cursor: pointer;` to the `:hover` css style for the card?